### ISSUE TITLE
Split JS out of JVM

### ISSUE THREAD

**Issue by kezz** [KyoriPowered/adventure-webui#41]
By running the project with the JS and JVM modules separate, we would be able to better utilise the Kotlin/JS environment with a node-like system, or similar.

**Comment by MiniDigger** [comment]
hot reload for JS pls

**Comment by kezz** [comment]
There's two ways we can go about this; having the front-end be a static webpage or as a node/smth application. I'm leaning towards the former, as it allows for super easy hot reloading and a decoupling of the front and back ends of the software would be nice.

**Comment by MiniDigger** [comment]
yeah we have no need for node since we got a backend already, just a normal static/spa style project
